similarity-data-rest/pom.xml

```diff
@@ -27,7 +27,7 @@
   </parent>
 
   <properties>
-    <coverage.instructionRatio>1.00</coverage.instructionRatio>
+    <coverage.instructionRatio>0.93</coverage.instructionRatio>
```

Comment:
Maybe we should write a test for that? At least one which just checks that `this.matchingNotificationManager.saveIncomingMatches()` is called at some point